scripts/RoboFile.php

```diff
@@ -0,0 +1,44 @@
+<?php
+/**
+ * This is project's console commands configuration for Robo task runner.
+ *
+ * @see http://robo.li/
+ */
+class RoboFile extends \Robo\Tasks
+{
+  const PROJECT_ROOT = __DIR__ . '/../';
```

Comment:
`__DIR__ . '/foo'`; is fine but `dirname(__FILE__)` would have a fit in a constant declaration in earlier versions